ocp3/profiles/coreos-ncp.profile

```diff
@@ -0,0 +1,62 @@
+documentation_complete: true
+
```

Comment:
Why do you change OCP 3? Isn't the request for multiple benchmarks about OCP 4?

Reply:
Because OCP4 stuff is in different PR. This is just an example, I'll get rid of it before merging.